editor/project_manager.cpp

```diff
@@ -2520,27 +2529,32 @@ ProjectManager::ProjectManager() {
 	tree_vb->set_custom_minimum_size(Size2(120, 120));
 	local_projects_hb->add_child(tree_vb);
 
+	const int btn_h_separation = int(6 * EDSCALE);
+
 	Button *open = memnew(Button);
```

Comment:
For consistency, does it make sense to assign this and the remaining buttons directly? Or am I missing something?
`open_btn = memnew(Button);`
`run_btn = memnew(Button);`
`rename_btn = memnew(Button);`
`erase_missing_btn = memnew(Button);`
`erase_btn = memnew(Button);`